DTS-STN/Alpha-Site
# Added script tags for staging

# Description

[Add script tags to application document as required by documentation](https://trello.com/c/KIrWaU2e/246-246-add-script-tags-to-application-document-as-required-by-documentation)

I added script tags that are going to be used for analytics. For now, it looks like it's not collecting anything. I think it's related to the other analytics tasks. 

## Acceptance Criteria

It shouldn't crash and there should be an adobe script tag in the head of a page and also a script tag right before the end of the body tag with _satellite.pageBottom();

## Test Instructions

1. Run the code and see if the two scripts are there in the head and right before the end of body
2. It shouldn't crash saying the _satellite.pageBottom() function doesn't exist

## Help Requested

I read the documentation and put what they said, but some things weren't clear, so if you guys check the documentation, any feedback is welcomed!

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
